pkg/expapi/v1/defaults.go

```diff
@@ -16,4 +16,16 @@ limitations under the License.
 
 package v1
 
-func addDefaultingFuncs() {}
+import (
+	"k8s.io/kubernetes/pkg/api"
+)
+
+func addDefaultingFuncs() {
+	api.Scheme.AddDefaultingFuncs(
+		func(obj *Scaler) {
+			if obj.Spec.Replicas == nil {
+				obj.Spec.Replicas = new(int)
```

Comment:
Yes, but it could be handy for updates so that the same scheme is used in RC and Scale(r).